.github/workflows/python-client.yml

```diff
@@ -0,0 +1,48 @@
+name: Python client checks, package build and deployment
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - develop
+      - master
+    paths:
+      - ./client/
+      - .github/workflows/python-client.yml
+  pull_request:
+    paths:
+      - ./client/
+      - .github/workflows/python-client.yml
+
+jobs:
+  lint:
+    name: Linting
+    runs-on: ubuntu-latest
+    steps:
+      - name: Clone
+        uses: actions/checkout@v3
+      - run: pip install flake8
+      - name: Flake8 lint Python code
+        run: (cd client && find src/ -type f -name '*.py' -exec flake8 --max-line-length=120 '{}' '+')
+
+  mypy:
+    name: Type checking
+    runs-on: ubuntu-latest
+    steps:
+      - name: Clone
+        uses: actions/checkout@v3
+      - run: pip install mypy
+      - name: Mypy type checking
+        run: (cd client && mypy src/)
+
+  packaging:
+    needs: [lint, mypy]
+    name: Build, test and deploy the Python package
+    uses: LedgerHQ/ledger-app-workflows/.github/workflows/reusable_pypi_deployment.yml@pypi_deployment
+    with:
+      package_directory: "client/"
+      stable_deployment: ${{ github.ref == 'refs/heads/master' }}
+      check_changelog_version: true
+      publish: ${{ github.event_name == 'push' }}
+    secrets:
+      pypi_token: ${{ github.ref == 'refs/heads/master' && secrets.PYPI_PUBLIC_API_TOKEN  || secrets.TEST_PYPI_PUBLIC_API_TOKEN }}
```

Comment:
Not sure I understand this line ? Won't this evaluate a true/false boolean ?

Reply:
It seems it is an accepted 'fake' ternary conditional operator on GitHub actions.
References:
- https://github.com/actions/runner/issues/409#issuecomment-752775072
- https://github.com/orgs/community/discussions/26738#discussioncomment-3836389

Reply:
(but yeah it needs to be properly deployed to assert it works, as for now current CI never tried to push a package so far)